### PR TITLE
feat: migrate OpenAI translation model to gpt-5.6-luna

### PR DESCRIPTION
**Pull request by Vladyslav-Soldatenko**
## Description

Switches the AI-translation feature from `gpt-5-mini` to `gpt-5.6-luna`.

## Type of Change

- [ ] New feature
- [ ] Bug fix
- [ ] Refactoring (no functional changes)
- [x] Documentation update
- [ ] Infrastructure / CI/CD
- [ ] Performance improvement

## Changes Made

- Swapped the OpenAI model constant in `OpenAiTranslationProvider.kt` from `ChatModel.GPT_5_MINI` to `ChatModel.GPT_5_6_LUNA` (confirmed this is a real named constant in the pinned `openai-java` 4.52.0 SDK, so no raw-string workaround was needed).
- Updated `README.md` to reference `GPT-5.6-luna` instead of `GPT-5-mini`, and dropped the "a few cents" cost claim since I have no verified pricing for the new model.

## How to Test

1. Set an `OPENAI_API_KEY` and use the "let AI draft translations" flow in the app.
2. Confirm a translation is returned successfully via the new model.

## Checklist

- [x] Code follows the project's style guidelines
- [x] Self-reviewed the code for obvious errors
- [ ] Added or updated tests where applicable, including edge cases
- [x] Existing tests pass locally
- [x] Updated documentation if needed
- [x] No duplication introduced. I searched the existing code for similar behavior and extracted similar parts to be reused
- [x] No new warnings or supressions introduced. If something like this has to be introduced - get approve from maintainer first

## Related Issues

N/A

---
_Generated by [Claude Code](https://claude.ai/code/session_01ND8LMTZKbe5n5cmh7z9izE)_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ ProcrastiLearn is an Android (Kotlin + Jetpack Compose) app that turns distracti
 ## Features
 - 📱 Overlay gate on chosen apps using Accessibility + overlay permissions.
 - 🧠 Spaced repetition via FSRS with daily caps.
-- ➕ Add words manually or let AI draft translations (prompt is editable). Currently GPT-5-mini is used, which means that you can add hundreds of words for a few cents.
+- ➕ Add words manually or let AI draft translations (prompt is editable). Currently GPT-5.6-luna is used.
```
Comment (Vladyslav-Soldatenko) [inline README.md]: restore ", which means that you can add hundreds of words for a few cents."

Comment (Vladyslav-Soldatenko) [reply README.md]: Restored in cc2a9e2.

---
_Generated by [Claude Code](https://claude.ai/code)_
